File: content/developer-tools/developer-tools_images_03/sdk/CreateVpc.java
import com.aliyuncs.DefaultAcsClient;
import com.aliyuncs.IAcsClient;
import com.aliyuncs.exceptions.ClientException;
import com.aliyuncs.exceptions.ServerException;
import com.aliyuncs.profile.DefaultProfile;
import com.google.gson.Gson;
import java.util.*;
import com.aliyuncs.vpc.model.v20160428.*;

public class CreateVpc {

    public static void main(String[] args) {
        DefaultProfile profile = DefaultProfile.getProfile("ap-northeast-1", "<accessKeyId>", "<accessSecret>");
        IAcsClient client = new DefaultAcsClient(profile);

        CreateVpcRequest request = new CreateVpcRequest();
        request.setRegionId("ap-northeast-1");
        request.setCidrBlock("172.16.0.0");
        request.setVpcName("sdk_vpc");
        request.setDescription("from openAPI sdk");

        try {
            CreateVpcResponse response = client.getAcsResponse(request);
            System.out.println(new Gson().toJson(response));
        } catch (ServerException e) {
            e.printStackTrace();
        } catch (ClientException e) {
            System.out.println("ErrCode:" + e.getErrCode());
            System.out.println("ErrMsg:" + e.getErrMsg());
            System.out.println("RequestId:" + e.getRequestId());
        }

    }
}

File: content/developer-tools/developer-tools_images_03/sdk/CreateVpc.py
#!/usr/bin/env python
#coding=utf-8

from aliyunsdkcore.client import AcsClient
from aliyunsdkcore.acs_exception.exceptions import ClientException
from aliyunsdkcore.acs_exception.exceptions import ServerException
from aliyunsdkvpc.request.v20160428.CreateVpcRequest import CreateVpcRequest

client = AcsClient('<accessKeyId>', '<accessSecret>', 'ap-northeast-1')

request = CreateVpcRequest()
request.set_accept_format('json')

request.set_CidrBlock("172.16.0.0/12")
request.set_VpcName("sdk_vpc")
request.set_Description("from openAPI sdk")

response = client.do_action_with_exception(request)
# python2:  print(response) 
print(str(response, encoding='utf-8'))
